dex/networks/eth/doc.go

```diff
@@ -0,0 +1,13 @@
+//go:build gpl
```

Comment:
The way I see the golang project structuring their docs is with the license at the top, the `go:build` and `+build` lines separated from other comments by white space, and then the `package` docs.  So three chunks of comments.  https://github.com/golang/text/commit/8f690f22cf1c026c950adddf3d45258bfd0912f0#diff-e03ec920b0cdec012a5f833c15db00c9d4ce59332ac08e9a90bd5e32e9ad16fb

When I run my Go 1.17 `go fmt` on these packages, it even separates the `go:build` parts from the rest of the comments for me.  Essentially, the file should look like this:

```go
// This code is available on the terms of the project LICENSE.md file,
// also available online at https://blueoakcouncil.org/license/1.0.0.
//
// This package also imports go-ethereum code and so carries the burden of
// go-ethereum's GNU Lesser General Public License.

//go:build gpl
// +build gpl

// Package eth implements methods to work with ethereum swap contracts and
// transactions. The code is unique as it requires a more restrictive liscense
// and connot be used in proprietary software.
package eth
```

Reply:
I have been relying on my editor (gvim...) plugin to format, which is probably behind the times. Will format correctly.

Reply:
`go fmt` behaviour actually change from go 1.16 -> go 1.17, which is why CI's version of golangci-lint isn't catching this.  Locally it changes though.

Reply:
editor plugin (go-vim) was using gopls by default, changed to gofmt

Reply:
Hopefully fixed.